docs/machine-learning/resources/learners.md

```diff
@@ -0,0 +1,67 @@
+# Learning Algorithms
+The following table will help you understand the different learning algorithms (learners) included in ML.NET and the [machine learning tasks](tasks.md) that they can be used for. Click on the learner name for more in-depth explanations on each of these learning methods.
+
+> [!NOTE]
+> ML.NET is currently in Preview. Not all learning algorithms are currently supported. To submit a request for a certain learner, please open an issue in the [dotnet/machinelearning repository](https://github.com/dotnet/machinelearning/issues).
+
+| Learner Name | Binary classification | Multi-class classification | Regression | Ranking | Clustering | Anomaly Detection |
+| --- |:---:|:---:|:---:|:---:|:---:|:---:|
+| [Averaged Perceptron](#averaged-perceptron) |:heavy_check_mark:||||||
+| [FastForest](#fastforest) |:heavy_check_mark:||:heavy_check_mark:||||
+| [FastTree](#fasttree)|:heavy_check_mark:||:heavy_check_mark:|:heavy_check_mark:|||
+| [Generalized Additive Models](#generalized-additive-models) |:heavy_check_mark:||:heavy_check_mark:||||
+| [K-Means++](#k-means) |||||:heavy_check_mark:||
+| [Linear SVM](#linear-svm) |:heavy_check_mark:||||||
+| [Logistic Regressor](#logistic-regressor) |:heavy_check_mark:|:heavy_check_mark:|||||
+| [Naïve Bayes](#naïve-bayes) ||:heavy_check_mark:|||||
```

Comment:
Too many dots above the "i"? :) In our code and in majority of textbooks it's "Naive", which is also easier to search for users.